src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -2937,13 +2937,13 @@ self =>
      *  ObjectDef       ::= Id ClassTemplateOpt
      *  }}}
      */
-    def objectDef(start: Offset, mods: Modifiers): ModuleDef = {
+    def objectDef(start: Offset, mods: Modifiers, isPackageObject: Boolean = false): ModuleDef = {
       in.nextToken()
       val nameOffset = in.offset
       val name = ident()
       val tstart = in.offset
       atPos(start, if (name == nme.ERROR) start else nameOffset) {
-        val template = templateOpt(mods, name, NoMods, Nil, tstart)
+        val template = templateOpt(mods, if  (isPackageObject) nme.PACKAGEkw else name, NoMods, Nil, tstart)
```

Comment:
scalafmt where are you? SIP-12 where are you? There's always either too many spaces or not enough spaces around a paren. `if ( isPO )` ofc. Also ofc we need API for `name.orElsePackageIf(b)`. But I'd be satisfied with deleting one space, if my new computer glasses worked.

Reply:
but! who doesn't like more          space!